src/Phinx/Config/Config.php

```diff
@@ -294,6 +306,58 @@ public function getTemplateClass()
      }
 
     /**
+     * Get the version order.
+     *
+     * @return string
+     */
+    public function getVersionOrder()
+    {
+        return $this->values['version_order'];
+    }
+
+    /**
+     * Set the version order.
+     * @param string $versionOrder The Version Order to set.
+     */
+    public function setVersionOrder($versionOrder = null)
+    {
+        if (!isset($versionOrder)) {
+            $versionOrder = isset ( $this->values['version_order'] ) ? $this->values['version_order'] : null;
+        }
+
+        if (!isset($versionOrder)) {
+            // if the configuration value is missing we use the default version order (creation time)
+            $this->values['version_order'] = self::VERSION_ORDER_CREATION_TIME;
```

Comment:
so I should remove the setter and replace the getter with that? Where should the guard go to? Having an invalid configuration version order (because of a typo, for instance) might lead to unexpected behaviour